build.savant

```diff
@@ -47,18 +48,18 @@ project(group: "org.savantbuild", name: "savant-dependency-management", version:
 }
 
 // Plugins
-dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.6")
-java = loadPlugin(id: "org.savantbuild.plugin:java:2.0.0-RC.6")
-javaTestNG = loadPlugin(id: "org.savantbuild.plugin:java-testng:2.0.0-RC.6")
-idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.6")
-release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
+def dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.7")
+def java = loadPlugin(id: "org.savantbuild.plugin:java:2.0.0-RC.6")
+def javaTestNG = loadPlugin(id: "org.savantbuild.plugin:java-testng:2.0.0-RC.6")
+def idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.7")
+def release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
 
 // Plugin settings
 java.settings.javaVersion = "17"
 javaTestNG.settings.javaVersion = "17"
 idea.settings.moduleMap = [
-    "org.savantbuild:savant-utils:2.0.0-RC.6"  : "savant-utils",
-    "org.savantbuild:savant-version:2.0.0-RC.6": "savant-version"
+    ("org.savantbuild:savant-utils:${savantVersion}".toString())  : "savant-utils",
```

Comment:
I don't think you need to call `toString()` or wrap this in a paren. Am I missing something?

Reply:
Reverted after talking with Daniel. I did this mainly, in this case, to keep IntelliJ from complaining about GStrings but there are other ways to accomplish that, see:
- https://github.com/savant-build/savant-ij/pull/2